examples/example/example.ino

```diff
@@ -28,7 +28,7 @@ byte syncWord[2] = {199, 10};
 bool packetWaiting;
 
 unsigned long lastSend = 0;
-unsigned int sendDelay = 5000;
```

Comment:
Removing of the send delay uses the frequency excessively. I would recommend not broadcasting continuously.